Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM debian
 
-RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
-RUN apt-get update
-RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install postgresql
+RUN apt update
+RUN DEBIAN_FRONTEND="noninteractive" apt -y install tzdata
+RUN DEBIAN_FRONTEND="noninteractive" apt -y install postgresql
```

Comment:
May be considered best practices to merge these `RUN` statements:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run